### ISSUE TITLE
5.4.1 breaks build

### ISSUE THREAD

**Issue by johnsonlin** [ionic-team/cordova-plugin-ionic#219]
**cordova plugin ls output:**
```
cordova-plugin-app-launcher 0.4.0 "Launcher"
cordova-plugin-apprate 1.4.0 "AppRate"
cordova-plugin-appsflyer-sdk 4.4.15 "AppsFlyer"
cordova-plugin-contacts 3.0.1 "Contacts"
cordova-plugin-device 2.0.2 "Device"
cordova-plugin-dialogs 2.0.1 "Notification"
cordova-plugin-email-composer 0.9.2 "EmailComposer"
cordova-plugin-file 6.0.1 "File"
cordova-plugin-file-transfer 1.7.1 "File Transfer"
cordova-plugin-fingerprint-aio 1.7.0 "FingerprintAllInOne"
cordova-plugin-globalization 1.11.0 "Globalization"
cordova-plugin-google-analytics 1.8.6 "Google Universal Analytics Plugin"
cordova-plugin-inappbrowser 3.0.0 "InAppBrowser"
cordova-plugin-ionic 5.3.0 "cordova-plugin-ionic"
cordova-plugin-ionic-keyboard 2.1.3 "cordova-plugin-ionic-keyboard"
cordova-plugin-ionic-webview 2.4.1 "cordova-plugin-ionic-webview"
cordova-plugin-nativestorage 2.3.2 "NativeStorage"
cordova-plugin-network-information 2.0.1 "Network Information"
cordova-plugin-screen-orientation 3.0.1 "Screen Orientation"
cordova-plugin-secure-storage 2.6.8 "SecureStorage"
cordova-plugin-splashscreen 5.0.2 "Splashscreen"
cordova-plugin-statusbar-sparkle 2.4.3 "StatusBar"
cordova-plugin-whitelist 1.3.3 "Whitelist"
cordova-plugin-x-socialsharing 5.4.5 "SocialSharing"
dynatrace-cordova-plugin 7.2.3 "Dynatrace"
es6-promise-plugin 4.2.2 "Promise"
mx.ferreyra.callnumber 0.0.2 "Cordova Call Number Plugin"
@ionic-enterprise/apple-payment-pass 1.0.2 "@ionic-enterprise/apple-payment-pass"
```

**ionic info output:**
```
Ionic:

   Ionic CLI          : 5.2.1 (C:\Users\xxx\AppData\Roaming\npm\node_modules\ionic)
   Ionic Framework    : ionic-angular 3.9.5
   @ionic/app-scripts : 3.2.3

Cordova:

   Cordova CLI       : 9.0.3 (cordova-lib@9.0.2)
   Cordova Platforms : android 7.1.4
   Cordova Plugins   : cordova-plugin-ionic 5.3.0, cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 2.4.1, (and 21
 other plugins)

Utility:

   cordova-res : not installed
   native-run  : 0.2.2

System:

   Android SDK Tools : 26.1.1 (C:\Users\xxx\AppData\Local\Android\Sdk)
   NodeJS            : v12.2.0 (C:\Program Files\nodejs\node.exe)
   npm               : 6.9.0
   OS                : Windows 10
```

**Describe the bug**
Latest npm release (5.4.1) contains typing errors and breaks build

**To Reproduce**
1. `npm i cordova-plugin-ionic@latest`
2. `ionic serve`
3. See error

**Expected behavior**
Build should pass and serve starts 

**Screenshots**

**Additional context**
```
[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 4 

            '>' expected. 



       L3:  configure(config: IDeployConfig): Promise<void>;

       L4:  getConfiguration(): Promise<import("./IonicCordova").ICurrentConfig>;

       L5:  checkForUpdate(): Promise<import("./IonicCordova").CheckForUpdateResponse>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 4 

            Identifier expected. 



       L3:  configure(config: IDeployConfig): Promise<void>;

       L4:  getConfiguration(): Promise<import("./IonicCordova").ICurrentConfig>;

       L5:  checkForUpdate(): Promise<import("./IonicCordova").CheckForUpdateResponse>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 4 

            '{' or ';' expected. 



       L3:  configure(config: IDeployConfig): Promise<void>;

       L4:  getConfiguration(): Promise<import("./IonicCordova").ICurrentConfig>;

       L5:  checkForUpdate(): Promise<import("./IonicCordova").CheckForUpdateResponse>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 4 

            '=' expected. 



       L3:  configure(config: IDeployConfig): Promise<void>;

       L4:  getConfiguration(): Promise<import("./IonicCordova").ICurrentConfig>;

       L5:  checkForUpdate(): Promise<import("./IonicCordova").CheckForUpdateResponse>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 4 

            Expression expected. 



       L3:  configure(config: IDeployConfig): Promise<void>;

       L4:  getConfiguration(): Promise<import("./IonicCordova").ICurrentConfig>;

       L5:  checkForUpdate(): Promise<import("./IonicCordova").CheckForUpdateResponse>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 5 

            '>' expected. 



       L4:  getConfiguration(): Promise<import("./IonicCordova").ICurrentConfig>;

       L5:  checkForUpdate(): Promise<import("./IonicCordova").CheckForUpdateResponse>;

       L6:  downloadUpdate(progress?: CallbackFunction<number>): Promise<boolean>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 5 

            Identifier expected. 



       L4:  getConfiguration(): Promise<import("./IonicCordova").ICurrentConfig>;

       L5:  checkForUpdate(): Promise<import("./IonicCordova").CheckForUpdateResponse>;

       L6:  downloadUpdate(progress?: CallbackFunction<number>): Promise<boolean>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 5 

            '{' or ';' expected. 



       L4:  getConfiguration(): Promise<import("./IonicCordova").ICurrentConfig>;

       L5:  checkForUpdate(): Promise<import("./IonicCordova").CheckForUpdateResponse>;

       L6:  downloadUpdate(progress?: CallbackFunction<number>): Promise<boolean>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 5 

            '=' expected. 



       L4:  getConfiguration(): Promise<import("./IonicCordova").ICurrentConfig>;

       L5:  checkForUpdate(): Promise<import("./IonicCordova").CheckForUpdateResponse>;

       L6:  downloadUpdate(progress?: CallbackFunction<number>): Promise<boolean>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 5 

            Expression expected. 



       L4:  getConfiguration(): Promise<import("./IonicCordova").ICurrentConfig>;

       L5:  checkForUpdate(): Promise<import("./IonicCordova").CheckForUpdateResponse>;

       L6:  downloadUpdate(progress?: CallbackFunction<number>): Promise<boolean>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 9 

            '>' expected. 



       L8:  reloadApp(): Promise<boolean>;

       L9:  ncOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 9 

            Identifier expected. 



       L8:  reloadApp(): Promise<boolean>;

       L9:  cOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 9 

            '{' or ';' expected. 



       L8:  reloadApp(): Promise<boolean>;

       L9:  cOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 9 

            '=' expected. 



       L8:  reloadApp(): Promise<boolean>;

       L9:  cOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 9 

            Expression expected. 



       L8:  reloadApp(): Promise<boolean>;

       L9:  cOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 10 

            '>' expected. 



       L9:  sync(options: ISyncOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 10 

            Identifier expected. 



       L9:  sync(options: ISyncOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 10 

            '{' or ';' expected. 



       L9:  sync(options: ISyncOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 10 

            '=' expected. 



       L9:  sync(options: ISyncOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 10 

            Expression expected. 



       L9:  sync(options: ISyncOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 11 

            '>' expected. 



      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;

      L12:  deleteVersionById(versionId: string): Promise<boolean>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 11 

            Identifier expected. 



      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;

      L12:  deleteVersionById(versionId: string): Promise<boolean>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 11 

            '{' or ';' expected. 



      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;

      L12:  deleteVersionById(versionId: string): Promise<boolean>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 11 

            '=' expected. 



      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;

      L12:  deleteVersionById(versionId: string): Promise<boolean>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 11 

            Expression expected. 



      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;

      L12:  deleteVersionById(versionId: string): Promise<boolean>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 13 

            '>' expected. 



      L12:      deleteVersionById(versionId: string): Promise<boolean>;

      L13:      getVersionById(versionId: string): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 13 

            Identifier expected. 



      L12:      deleteVersionById(versionId: string): Promise<boolean>;

      L13:      getVersionById(versionId: string): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 13 

            '{' or ';' expected. 



      L12:      deleteVersionById(versionId: string): Promise<boolean>;

      L13:      getVersionById(versionId: string): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 13 

            '=' expected. 



      L12:      deleteVersionById(versionId: string): Promise<boolean>;

      L13:      getVersionById(versionId: string): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 13 

            Expression expected. 



      L12:      deleteVersionById(versionId: string): Promise<boolean>;

      L13:      getVersionById(versionId: string): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 4 

            Generic type 'Promise<T>' requires 1 type argument(s). 



       L3:  configure(config: IDeployConfig): Promise<void>;

       L4:  getConfiguration(): Promise<import("./IonicCordova").ICurrentConfig>;

       L5:  checkForUpdate(): Promise<import("./IonicCordova").CheckForUpdateResponse>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 4 

            A parameter initializer is only allowed in a function or constructor implementation. 



       L3:  configure(config: IDeployConfig): Promise<void>;

       L4:  getConfiguration(): Promise<import("./IonicCordova").ICurrentConfig>;

       L5:  checkForUpdate(): Promise<import("./IonicCordova").CheckForUpdateResponse>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 5 

            Generic type 'Promise<T>' requires 1 type argument(s). 



       L4:  getConfiguration(): Promise<import("./IonicCordova").ICurrentConfig>;

       L5:  checkForUpdate(): Promise<import("./IonicCordova").CheckForUpdateResponse>;

       L6:  downloadUpdate(progress?: CallbackFunction<number>): Promise<boolean>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 5 

            A parameter initializer is only allowed in a function or constructor implementation. 



       L4:  getConfiguration(): Promise<import("./IonicCordova").ICurrentConfig>;

       L5:  checkForUpdate(): Promise<import("./IonicCordova").CheckForUpdateResponse>;

       L6:  downloadUpdate(progress?: CallbackFunction<number>): Promise<boolean>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 9 

            Generic type 'Promise<T>' requires 1 type argument(s). 



       L8:  reloadApp(): Promise<boolean>;

       L9:  ions: ISyncOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | u

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 9 

            A parameter initializer is only allowed in a function or constructor implementation. 



       L8:  reloadApp(): Promise<boolean>;

       L9:  cOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 9 

            Object is possibly 'undefined'. 



       L8:  reloadApp(): Promise<boolean>;

       L9:  cOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 9 

            The right-hand side of an arithmetic operation must be of type 'any', 'number' or an enum type. 



       L8:  reloadApp(): Promise<boolean>;

       L9:  cOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 10 

            Generic type 'Promise<T>' requires 1 type argument(s). 



       L9:  sync(options: ISyncOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 10 

            A parameter initializer is only allowed in a function or constructor implementation. 



       L9:  sync(options: ISyncOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 10 

            Duplicate identifier 'ISnapshotInfo'. 



       L9:  sync(options: ISyncOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 10 

            Object is possibly 'undefined'. 



       L9:  sync(options: ISyncOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 10 

            The right-hand side of an arithmetic operation must be of type 'any', 'number' or an enum type. 



       L9:  sync(options: ISyncOptions, progress?: CallbackFunction<number>): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 11 

            Generic type 'Promise<T>' requires 1 type argument(s). 



      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;

      L12:  deleteVersionById(versionId: string): Promise<boolean>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 11 

            A parameter initializer is only allowed in a function or constructor implementation. 



      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;

      L12:  deleteVersionById(versionId: string): Promise<boolean>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 11 

            Duplicate identifier 'ISnapshotInfo'. 



      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;

      L12:  deleteVersionById(versionId: string): Promise<boolean>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 11 

            Subsequent variable declarations must have the same type. Variable 'ISnapshotInfo' must be of type 'number', 

            but here has type 'boolean'. 



      L10:  getCurrentVersion(): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;

      L11:  getAvailableVersions(): Promise<import("./IonicCordova").ISnapshotInfo[]>;

      L12:  deleteVersionById(versionId: string): Promise<boolean>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 13 

            Generic type 'Promise<T>' requires 1 type argument(s). 



      L12:      deleteVersionById(versionId: string): Promise<boolean>;

      L13:      getVersionById(versionId: string): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 13 

            A parameter initializer is only allowed in a function or constructor implementation. 



      L12:      deleteVersionById(versionId: string): Promise<boolean>;

      L13:      getVersionById(versionId: string): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 13 

            Duplicate identifier 'ISnapshotInfo'. 



      L12:      deleteVersionById(versionId: string): Promise<boolean>;

      L13:      getVersionById(versionId: string): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 13 

            Object is possibly 'undefined'. 



      L12:      deleteVersionById(versionId: string): Promise<boolean>;

      L13:      getVersionById(versionId: string): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;



[01:34:45]  typescript: node_modules/cordova-plugin-ionic/dist/index.d.ts, line: 13 

            The right-hand side of an arithmetic operation must be of type 'any', 'number' or an enum type. 



      L12:      deleteVersionById(versionId: string): Promise<boolean>;

      L13:      getVersionById(versionId: string): Promise<import("./IonicCordova").ISnapshotInfo | undefined>;
```

**Comment by gnrmarcel** [comment]
have the same Problem on MacOS …

**Comment by nphyatt** [comment]
Yeah sorry this was an issue with typescript compatibility with typescript 2.x I think it affected 5.4.0-2 fixed in 5.4.3.
